### ISSUE TITLE
Indexing fails when document contains field named status

### ISSUE THREAD

**Issue by kbrabrand** [phoob/elasticraft#2]
If the transformer returns an associative array containing a property named `status` the indexing fails (silently) because the field is expected to be an object. Most likely caused by `status` being a protected field name. 

Example data from transformer:

```php
[
  "id" => 123,
  "status" => "ok"
]
```

Error message returned from elastic:

```php
[
  "type" => "mapper_parsing_exception",
  "reason" => "object mapping for [status] tried to parse field [status] as object, but found a concrete value"
]
```

**Comment by phoob** [comment]
@kbrabrand did you try to delete the index? I'm pretty sure "status" is not a reserved field name. But, if it was previously used in the index, and the field value was then an object, and you're now trying to use the same field with a string value, this could happen...

**Comment by phoob** [comment]
@kbrabrand basically, whenever you edit a field definition in a transformer, it is probably a good idea to recreate the elastic index...

**Comment by kbrabrand** [comment]
You are absolutely right. Delete now, and it worked. Would probably be a good idea to show the errors one way or another. Right now it fails silently.

**Comment by phoob** [comment]
Hmm.. I guess the right thing to do would be to refactor and create a task to perform the indexing.. and let that fail with an error message. Agree?

**Comment by kbrabrand** [comment]
That sounds right to me

**Comment by phoob** [comment]
Fixed in 0.1.0
